api/src/clojure/org/akvo/flow_api/datastore/form_instance.clj

```diff
@@ -103,9 +104,14 @@
   [_ response-str]
   response-str)
 
+(def ^DateFormat sdf (SimpleDateFormat. "dd-MM-yyyy HH:mm:ss z"))
```

Comment:
More useful background info on the issues behind `SimpleDateFormat` -  http://stackoverflow.com/a/4021932